static/sass/_settings.scss

```diff
@@ -4,5 +4,5 @@ $color-accent: $ubuntu-orange;
 $color-suru-start: #2c001e;
 $color-suru-middle: #772953;
 $color-suru-end: #e95420;
-$breakpoint-navigation-threshold: 600px;
+$breakpoint-navigation-threshold: 620px;
```

Comment:
Shall we add a comment here about base_layout beeing in sync as well?

Reply:
```suggestion
$breakpoint-navigation-threshold: 620px; // keep this in sync with global nav config in base_layout.html
```